docs/src/content/en/docs/workflows/overview.mdx

```diff
@@ -21,7 +33,10 @@ You create workflows by:
 
 This structure provides full type safety and runtime validation, ensuring data integrity across the entire workflow.
 
-> **📹 Watch**:  → An introduction to workflows, and how they compare to agents [YouTube (7 minutes)](https://youtu.be/0jg2g3sNvgw)
+### Visual testing
```

Comment:
Screenshot would be cool, maybe recreate the real-world example you laid out above to demonstrate a sufficiently complex workflow?

Reply:
What is the motivation behind this section by the way?